src/routes/boot.ts

```diff
@@ -515,14 +533,19 @@ const loggedInBoot = async ({
         canSubmitArticle: user.reputation >= submitArticleThreshold,
         isTeamMember,
         isPlus,
-        plusProvider,
         language: user.language || undefined,
         image: mapCloudinaryUrl(user.image),
         cover: mapCloudinaryUrl(user.cover),
         defaultFeedId: isPlus ? user.defaultFeedId : null,
         flags: {
           showPlusGift: Boolean(user?.flags?.showPlusGift),
         },
+        subscriptionFlags: {
+          ...includeProperties(user.subscriptionFlags!, [
+            'provider',
+            'appAccountToken',
+          ]),
+        },
```

Comment:
Let's just make it static, it is faster and easier to see what is going on, `excludeProperties` above is something legacy so we never touched it. 
```suggestion
        subscriptionFlags: {
          provider: user.subscriptionFlags?.provider,
          appAccountToken: user.subscriptionFlags?.appAccountToken,
        },
```

Reply:
I created a new function that's the opposite of exclude, so that it just includes the properties.
But I'll make it static :)